webextensions/api/webRequest.json

```
@@ -1505,6 +1505,27 @@
             }
           },
           "details": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "≤58"
```

See: https://github.com/caugner/chromium-releases/blame/v58/extensions/common/api/web_request.json

```suggestion
                  "version_added": "18"
```

Also added in Chrome 18:

- challenger
- frameId
- isProxy
- parentFrameId
- realm
- requestId
- responseHeaders
- scheme
- tabId
- timeStamp
- url

Chrome 43:
- statusCode
- statusLine

Chrome 44:
- type

---

To be added separately, as this PR is big enough